components/new-contribution-flow/utils.js

```diff
@@ -45,3 +48,42 @@ export const taxesMayApply = (collective, host, tier) => {
     return LibTaxes.getVatOriginCountry(tier.type, hostCountry, country);
   }
 };
+
+const DEFAULT_PRESETS = [0, 500, 1000, 1500, 5000];
+const DEFAULT_FUNDS_PRESETS = [100000, 200000, 500000, 1000000];
+
+export const getTierPresets = (tier, collectiveType) => {
```

Comment:
This is a good idea, maybe we can even put it in the general `utils` folder and use it elsewhere, since the update recurring contribution page also fetches tier presets

Reply:
Good call, let's move that to `lib/tier-utils.js`